scene/3d/navigation_agent_3d.h

```diff
@@ -153,10 +153,10 @@ class NavigationAgent3D : public Node {
 
 	Vector3 get_next_location();
 
-	const Vector<Vector3> &get_nav_path() const;
+	const Vector<Vector3> &get_current_navigation_path() const;
 
-	int get_nav_path_index() const {
-		return nav_path_index;
+	int get_navigation_path_index() const {
```

Comment:
```suggestion
	int get_current_navigation_path_index() const {
```

For consistency with `get_current_navigation_path`.